idseq_pipeline/commands/common.py

```diff
@@ -623,3 +609,16 @@ def upload_version_tracker(source_file, output_name, reference_version_number, o
 
 def env_set_if_blank(key, value):
     os.environ[key] = os.environ.get(key, value)
+
+def validate_taxid_lineage(taxid_lineage, hit_taxid, hit_level):
+    # Take the taxon lineage and mark meaningless calls with fake taxids.
+    # For each hit, the fake taxids below the meaningful hit level should depend
+    # only on: (a) the taxonomic level (b) the taxid of the meaningful hit
+    cleaned_taxid_lineage = []
+    for i in range(len(taxid_lineage)):
```

Comment:
If you run pylint on this, it will tell you to replace this line and the next with
```
    for i, taxid in enumerate(taxid_lineage)
```